content/en/docs/tasks/administer-cluster/reserve-compute-resources.md

```diff
@@ -23,12 +22,10 @@ compute resources for system daemons. Kubernetes recommends cluster
 administrators to configure 'Node Allocatable' based on their workload density
 on each node.
 
-## {{% heading "prerequisites" %}}
```

Comment:
I suggest keeping the heading ( `{{% heading "prerequisites" %}}` ) . [Currently](https://deploy-preview-45771--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/reserve-compute-resources/), its removal is leading to the introductory and pre-requisite content to blend together, making it difficult to distinguish between them.